test/sqllogictest/role_create.slt

```diff
@@ -0,0 +1,148 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Loosely based on https://github.com/postgres/postgres/blob/master/src/test/regress/expected/create_role.out
+
+mode cockroach
+
+reset-server
+
+simple conn=mz_system,user=mz_system
+ALTER SYSTEM SET enable_ld_rbac_checks TO true;
+----
+COMPLETE 0
+
+simple conn=mz_system,user=mz_system
+ALTER SYSTEM SET enable_rbac_checks TO true;
+----
+COMPLETE 0
+
+simple conn=mz_system,user=mz_system
+CREATE ROLE regress_role_admin CREATEDB CREATECLUSTER CREATEROLE;
```

Comment:
Are those 3 attributes all the attributes that we support at this time? the list at https://www.postgresql.org/docs/current/sql-alterrole.html includes some very obviously Postgres-specific ones, but what about `LOGIN`, `NOLOGIN` (those came and went in previous Mz versions).